events/javaland/sessions.html.slim

```diff
@@ -0,0 +1,97 @@
+---
+layout: microsite-multi-page
+hero_class: event-javaland
+
+nav: 
+  - title: About JavaLand
+    url: '../about/'
+  - title: Our Speakers
+    url: '../speakers/'
+  - title: Our Sessions
+    url: '../sessions/'
+  - title: Further Information
+    url: '../info/'
+---
+
+- page.title = "JBoss at JavaLand - The Sessions"
+
+= define 'hero'
+  .row 
+    .large-25.column.center
+      h2 JBoss at JavaLand
+
+
+h2.divider JBoss Sessions at JavaLand
+p Some more interesting content for you to check out  
+
+table
```

Comment:
This table would be easier to maintain and create if it was in a separate Asciidoc file. However, given that this site is unlikely to change (much) it's probably not worth the re-factor. Same applies for the table on the other page.